lib/trebuchet/strategy/per_denomination.rb

```diff
@@ -0,0 +1,19 @@
+class Trebuchet::Strategy::PerDenomination < Trebuchet::Strategy::Base
+  include Trebuchet::Strategy::PerDenominationable
+
+  def set_range_from_options(options = {})
+    numerator = options['numerator'] || options[:numerator] || 0
+    denominator = options['denominator'] || options[:denominator] || 0
```

Comment:
`denominator` being 0 would cause exception, perhaps default it to 100?

Reply:
i think it's better to have the exception (and launch to nobody) than launch to an arbitrary ratio. i'm only choosing `0` here as this would be the standard `.to_i` result - not because it's meaningful (as you observed, the `super` call will `raise` anyway)